### DISCUSSION TITLE
Logo replacement not working

### DISCUSSION THREAD

**Discussion by pxlwrk** [mediacms-io/mediacms#107] [Q&A]
When I replace the logo image by a custom image (svg) it is not showing up on the page. It is loaded but its dimensions in the browser are 0x0 pixels. What am I doing wrong?

**Comment by Dan1ell** [comment]
Not sure why it's not displaying new svgs in an appropriate size, but for me, I modified my svg to the dimensions of the original as a workaround:
<svg width="339px" height="56px" viewBox="0 0 339 56" ...

**Reply by GeorgiDimitrov87** [reply]
Hi I have the same issue. I made a png file onto svg and named it logo_light.svg and logo_dark.svg and put it in static/images.
the size of the logo is 24px 18 px no matter what I change. --logo-height and 
--default-logo-height are the css variables I find but when I modify them, nothing happens. I modified the svg to the same size and removed the svg so I can use only the png but still nothing

**Comment by mgogoulos** [comment]
I've converted this into an issue - https://github.com/mediacms-io/mediacms/issues/316 so it can be further looked after.
Until then, I hope Dan1ell's suggestion above works

**Comment by guntube** [comment]
Hey, just wanted to update that this is still an issue I am finding.

**Comment by s0apye** [comment]
Hey @guntube. I have fixed the issue by going into /templates/config/templates/site.html.

You will have to use the png extensions. 

Delete the two lines for the svg extensions so it looks like this:


        lightMode:{
            img: "/static/images/new/logo_dark.png",
        },
        darkMode:{
            img: "/static/images/new/logo_light.png",

**Reply by mgogoulos** [reply]
Hi @s0apye , feel free to share a public portal if you have one, just to see how this change looks!

**Comment by jenny787** [comment]
I've replaced both svg andpng logos also, and no change.  What is the way to check?  reboot?
